src/js/angular/rest/monitoring.rest.service.js

```diff
@@ -0,0 +1,44 @@
+angular
+    .module('graphdb.framework.rest.monitoring.service', [])
+    .factory('MonitoringRestService', MonitoringRestService);
+
+MonitoringRestService.$inject = ['$http'];
+
+function MonitoringRestService($http) {
+    return {
+        monitorResources,
+        monitorGC,
+        monitorQuery,
+        deleteQuery,
+        getQueryCount,
+        checkAutocompleteStatus
+    };
+
+    function monitorResources() {
+        return $http.get('rest/monitor/resource');
```

Comment:
Constant for `/rest/monitor` maybe

Reply:
Done. Also some more forgotten once were extracted.